packages/utils/src/partial_ord.cairo

```diff
@@ -0,0 +1,47 @@
+/// PartialOrd implementation for tuple (u32, felt252).
+/// Required to sort leaf nodes in proof verification.
+pub impl PartialOrdTupleU64Felt252 of PartialOrd<(u64, felt252)> {
+    fn le(lhs: (u64, felt252), rhs: (u64, felt252)) -> bool {
+        let (a, _) = lhs;
+        let (b, _) = rhs;
+
+        if a <= b {
+            true
+        } else {
+            false
+        }
+    }
+
+    fn ge(lhs: (u64, felt252), rhs: (u64, felt252)) -> bool {
+        let (a, _) = lhs;
+        let (b, _) = rhs;
+
+        if a >= b {
+            true
+        } else {
+            false
+        }
+    }
+
+    fn lt(lhs: (u64, felt252), rhs: (u64, felt252)) -> bool {
```

Comment:
It is enough to implement just `lt` (see https://github.com/starkware-libs/cairo/blob/7a2cdc90b61587360e0eefbee5d6de9174ef686d/corelib/src/traits.cairo#L101)

Reply:
nice!